jdaviz/configs/default/plugins/collapse/collapse.vue

```diff
@@ -49,5 +49,33 @@
       @click:action="collapse"
     ></plugin-add-results>
 
+    <j-plugin-section-header v-if="export_enabled">Results</j-plugin-section-header>
+
+    <div style="display: grid; position: relative"> <!-- overlay container -->
+      <div style="grid-area: 1/1">
+        <div v-if="collapsed_spec_available && export_enabled">
+
+        <v-row>
+              <v-text-field
+              v-model="filename"
+              label="Filename"
+              hint="Export the latest collapsed spectrum."
+              :rules="[() => !!filename || 'This field is required']"
+              persistent-hint>
+              </v-text-field>
+          </v-row>
+
+          <v-row justify="end">
+            <j-tooltip tipid='plugin-collapse-save-fits'>
+              <v-btn color="primary" text @click="save_as_fits">Save as FITS</v-btn>
+
+            </j-tooltip>
+          </v-row>
+
+        </div>
```

Comment:
doesn't affect anything, but can you please fix the indentation here (and for the other plugin)?